### PR TITLE
Adding less variables for table head background color and table row h…

### PR DESCRIPTION
**Pull request by kappa-gooner**
Adding less variables for setting table head background color and table row background hover color

### CODE REVIEW

**Comment by kappa-gooner** [comment]
Fix for issue: #5706

File: components/style/themes/default.less [hunk 1]
```diff
@@ -252,6 +252,11 @@
 @spin-dot-size: 20px;
 @spin-dot-size-lg: 32px;
 
+// Table
+// --
+@table-head-background-color: @background-color-base;
+@table-row-background-hover :  @primary-1;
```
Comment (afc163) [inline components/style/themes/default.less]: How aboult `@table-header-bg` `@table-row-hover-bg`?

Comment (afc163) [inline components/style/themes/default.less]: updated at 92b8e85

Comment (leadream) [inline components/style/themes/default.less]: cool!

Comment (39Er) [inline components/style/themes/default.less]: that's cool, I need it too!

**Pull request merged**